apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -129,18 +129,26 @@ private void MaintainPool()
         /// </summary>
         /// <returns>A snapshot of statistics for this pool.</returns>
         public Statistics GetStatisticsSnapshot() =>
-            new Statistics(_targetedPools.ToArray().Select(p => p.Value.GetStatisticsSnapshot()).ToList().AsReadOnly());
-
+            new Statistics(_targetedPools.ToArray().Select(p => p.Value.GetStatisticsSnapshot()).ToList().AsReadOnly());
+
         /// <summary>
         /// Provides a snapshot of statistics for a database-specific pool.
```

Comment:
Can you please add here (and in all the overloads that receive just the DatabaseName) something like:
```suggestion
        /// Provides a snapshot of statistics for a database-specific pool.
        /// This is equivalent to calling <see cref="GetStatisticsSnapshot(SessionPoolKey)"> passing a session pool key
        /// with a null database role.
```